src/providers/cortex/trulens/providers/cortex/provider.py

```diff
@@ -21,7 +24,7 @@ class Cortex(LLMProvider):
     """
 
     endpoint: CortexEndpoint
-    snowflake_conn: SnowflakeConnection
+    snowflake_conn: Any
```

Comment:
is this just to make Pydantic happy cuz we could get either the connection object or stored_proc? 

If that's the case couldn't we still keep the type hints with sth like `type1 | type2`? (unsure what might be the type of _SNOWFLAKE_STORED_PROCEDURE_CONNECTION)

Reply:
No it's because the actual stored procedure connection is like a special class from a library that's not on pypi. Specifically it's built from this repo:
https://github.com/snowflakedb/Stored-Proc-Python-Connector